template_feed/Microsoft.DotNet.Common.ProjectTemplates.10.0/content/MSTest-CSharp/.template.config/template.json

```diff
@@ -238,7 +238,7 @@
       "actionId": "84C0DA21-51C8-4541-9940-6CA19AF04EE6",
       "id": "openInEditor",
       "args": {
-        "files": "1"
+        "files": "2"
```

Comment:
Is the index-based of the file from primaryOutputs?
I see some conditions in primaryOutputs which I don't understand what they are for, and what would happen if the index here is referring to something whose condition evaluates to false.

Reply:
https://github.com/dotnet/templating/wiki/Using-Primary-Outputs-for-Post-Actions#example-3---using-primary-output-indexes-in-post-action-arguments this is example that is very similar to what is happening here, primary outputs are files that can later be post processed by post actions. 

this post action has the same condition as the primary outputs so the indexing is correct.

Reply:
changed to instead remove the unnecessary mstest settings file